changelog.rst

```diff
@@ -2,11 +2,22 @@
 Changelog
 =========
 
+Changes in version 1.2.0
+------------------------
+- Added ability to explain command options
```

Comment:
Can you rephrase this? Best would be to mention the feature or new parameter name itself so that users can find it.